samples/event_manager_profiler/prj.conf

```diff
@@ -19,5 +19,5 @@ CONFIG_HEAP_MEM_POOL_SIZE=2048
 CONFIG_REBOOT=y
 
 # Configuration required by Event Manager profiler tracer
```

Comment:
Should we also rename the sample (for consistency) ?

Reply:
The test scenario name was renamed, but name of the sample was left unchanged.

Reply:
Good point made changes